src/api/app/components/workflow_run_detail_component.rb

```diff
@@ -22,4 +23,18 @@ def parse_payload(workflow_run)
   rescue JSON::ParserError
     workflow_run.request_payload
   end
+
+  # Old workflow run entries didn't store the configuration-related information
+  def workflow_configuration_data
+    configuration_source = workflow_run.workflow_configuration_url if workflow_run.workflow_configuration_url.present?
+    configuration_source ||= workflow_run.workflow_configuration_path if workflow_run.workflow_configuration_path.present?
```

Comment:
Sometimes the value is an empty string (""), so I need to check the presence and not the existence of a value, as
` "".present? => false`.

Reply:
You can also use
```
[workflow_run.workflow_configuration_url, workflow_run.workflow_configuration_path].presence.compact.first
```

Reply:
I assume this is what you tried to suggest:

```
[workflow_run.workflow_configuration_url, workflow_run.workflow_configuration_path].map(&:presence).compact.first
```

As your code could return an empty array as a result, because:

```
[workflow_run.workflow_configuration_url, workflow_run.workflow_configuration_path].presence
=> ["", ".obs/workflows.yml"]
```

Reply:
Pushed the one with the `map`.

Reply:
Nice